tock/tock/templates/_navigation.html

```diff
@@ -10,11 +10,24 @@
           <span>Projects</span>
         </a>
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if request.user.is_staff %}
+        <li>
+          <button class="
+            usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="side-nav-2">
+            <span>Reports</span>
+          </button>
+          <ul id="side-nav-2" class="usa-nav-submenu" aria-hidden="true">
```

Comment:
Same comment as above for the `side-nav-2`.  Choose another id.

Reply:
Good catch, Amy. Those are also bad IDs, so I've made them a bit more descriptive.

Reply:
@tbaxter-18f Did you push the commit?

Reply:
I did now :-)